features/admin.users.v1/components/wizard/add-user-wizard.tsx

```diff
@@ -927,6 +932,7 @@ export const AddUserWizard: FunctionComponent<AddUserWizardPropsInterface> = (
                     hiddenFields={ hiddenFields }
                     requestedPasswordOption={ wizardState &&
                     wizardState[ WizardStepsFormTypes.BASIC_DETAILS ]?.passwordOption }
+                    isUserstoreRequired={ isUserstoreRequired }
```

Comment:
How about directly passing `isSubOrganization()` here instead of introducing another state variable?

Reply:
This was introduced as we might be require this in the future for other conditions similar to the other fields